<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QTableWidget" name="processTableWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>1061</width>
        <height>531</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1061</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="KillBut">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>570</y>
        <width>84</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>kill</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <widget class="QChartView" name="cpuWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1151</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QChartView" name="netWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>1151</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QChartView" name="IOWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>1151</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cpu1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cpu2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="net1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>420</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Net1</string>
      </property>
     </widget>
     <widget class="QLabel" name="net2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLineLabel1">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>200</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLineLabel2">
      <property name="geometry">
       <rect>
        <x>592</x>
        <y>200</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="netLable1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>420</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="netLable2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="IoLable1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>640</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="IoLable2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>640</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Io1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>640</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Io2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>640</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="cpuInfoListWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>800</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="sysInfoListWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>800</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>259</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OS</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="ProcessBut">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>11</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResourcesBut">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>11</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Resources</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSysBUt">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>11</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SysInfo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
